core/assets/bundles/bundle_id_ID.properties

```diff
@@ -35,171 +35,175 @@ load.mod = Mod
-schematic.edit = Edit Schematic
+schematic.flip = [accent][[{0}][]/[accent][[{1}][]: Balik Skema
+schematic.saved = Skema telah disimpan.
+schematic.delete.confirm = Skema ini akan benar-benar dihapus.
+schematic.edit = Sunting Skema
 schematic.info = {0}x{1}, {2} blok
-schematic.disabled = [scarlet]Bagan dilarang[]\nAnda tidak diperbolehkan untuk menggunakan bagan di [accent]peta[] atau [accent]server ini.
+schematic.disabled = [scarlet]Skema dimatikan[]\nPenggunaan skema tidak diperbolehkan di [accent]peta[] atau [accent]server[] ini.
 schematic.tags = Tanda:
 schematic.edittags = Ubah Tanda
 schematic.addtag = Tambah Tanda
-schematic.texttag = Teks Tanda
-schematic.icontag = Ikon Tanda
+schematic.texttag = Tanda Teks
+schematic.icontag = Tanda Ikon
 schematic.renametag = Ubah Nama Tanda
-schematic.tagged = {0} tagged
-schematic.tagdelconfirm = Hapus tanda ini sepenuhnya?
+schematic.tagged = {0} ditandai
+schematic.tagdelconfirm = Ingin menghapus tanda ini?
 schematic.tagexists = Tanda tersebut sudah ada.
 
 stats = Statistik
 stats.wave = Gelombang Terkalahkan
 stats.unitsCreated = Unit Terbentuk
-stats.enemiesDestroyed = Musuh Terhancurkan
+stats.enemiesDestroyed = Musuh yang Dihancurkan
 stats.built = Jumlah Blok yang Dibangun
 stats.destroyed = Jumlah Blok Dihancurkan Musuh
-stats.deconstructed = Jumlah Blok Dihancurkan Pemain
+stats.deconstructed = Jumlah Blok yang Dihancurkan Pemain
 stats.playtime = Waktu Bermain
 
 globalitems = [accent]Bahan Keseluruhan
-map.delete = Apakah Anda yakin ingin menghapus peta "[accent]{0}[]"?
+map.delete = Apakah anda yakin ingin menghapus peta "[accent]{0}[]"?
 level.highscore = Nilai Tertinggi: [accent]{0}
 level.select = Pilih Level
 level.mode = Mode Permainan:
 coreattack = < Inti sedang diserang! >
-nearpoint = [[ [scarlet]TINGGALKAN TITIK JATUH SEGERA[] ]\npenghancuran akan terjadi
+nearpoint = [[ [scarlet]TINGGALKAN ZONA PENDARATAN SEGERA[] ]\npenghancuran akan terjadi
 database = Basis Data Inti
 database.button = Basis Data
 savegame = Simpan Permainan
 loadgame = Muat Permainan
-joingame = Bermain Bersama
-customgame = Permainan Modifikasi
+joingame = Gabung Permainan
+customgame = Permainan Kustom
 newgame = Permainan Baru
 none = <kosong>
 none.found = [lightgray]<tidak ditemukan>
 none.inmap = [lightgray]<tidak ada di peta>
 minimap = Peta Kecil
 position = Posisi
 close = Tutup
-website = Situs Jaringan
+website = Situs Web
 quit = Keluar
 save.quit = Simpan & Keluar
 maps = Peta
 maps.browse = Cari Peta
 continue = Lanjutkan
 maps.none = [lightgray]Peta tidak ditemukan!
-invalid = Tidak valid
+invalid = Tidak Valid
 pickcolor = Pilih Warna
 preparingconfig = Menyiapkan Konfigurasi
 preparingcontent = Menyiapkan Konten
 uploadingcontent = Mengunggah Konten
 uploadingpreviewfile = Mengunggah File Tinjauan
 committingchanges = Membuat Perubahan
 done = Selesai
-feature.unsupported = Perangkat Anda tidak mendukung fitur ini.
+feature.unsupported = Perangkat anda tidak mendukung fitur ini.
```

Comment:
"Anda" should always have its first letter capitalized according to Indonesian orthography general guide (_Pedoman Umum Ejaan Bahasa Indonesia_/PUEBI)

Reply:
Hmm, why not use Enhanced Indonesian Spelling (EYD) instead of it?